impl/sdl/sdl.odin

```diff
@@ -94,52 +94,52 @@ process_event :: proc(e: sdl.Event, state: ^SDL_State) {
 
 update_dt :: proc(state: ^SDL_State) {
     io := imgui.get_io();
-    freq := sdl.get_performance_frequency();
-    curr_time := sdl.get_performance_counter();
+    freq := sdl.GetPerformanceFrequency();
+    curr_time := sdl.GetPerformanceCounter();
     io.delta_time = state.time > 0 ? f32(f64(curr_time - state.time) / f64(freq)) : f32(1/60);
     state.time = curr_time;
 }
 
 update_mouse :: proc(state: ^SDL_State, window: ^sdl.Window) {
     io := imgui.get_io();
     mx, my: i32;
-    buttons := sdl.get_mouse_state(&mx, &my);
-    io.mouse_down[0] = state.mouse_down[0] || (buttons & u32(sdl.Mousecode.Left))   != 0;
-    io.mouse_down[1] = state.mouse_down[1] || (buttons & u32(sdl.Mousecode.Right))  != 0;
-    io.mouse_down[2] = state.mouse_down[2] || (buttons & u32(sdl.Mousecode.Middle)) != 0;
+    buttons := sdl.GetMouseState(&mx, &my);
+    io.mouse_down[0] = state.mouse_down[0] || (buttons & u32(sdl.BUTTON_LEFT))   != 0;
+    io.mouse_down[1] = state.mouse_down[1] || (buttons & u32(sdl.BUTTON_RIGHT))  != 0;
+    io.mouse_down[2] = state.mouse_down[2] || (buttons & u32(sdl.BUTTON_MIDDLE)) != 0;
     state.mouse_down[0] = false;
     state.mouse_down[1] = false;
     state.mouse_down[2] = false;
 
     // Set mouse pos if window is focused
     io.mouse_pos = imgui.Vec2{min(f32), min(f32)};
-    if sdl.get_keyboard_focus() == window {
+    if sdl.GetKeyboardFocus() == window {
         io.mouse_pos = imgui.Vec2{f32(mx), f32(my)};
     }
 
     if io.config_flags & .NoMouseCursorChange != .NoMouseCursorChange {
         desired_cursor := imgui.get_mouse_cursor();
         if(io.mouse_draw_cursor || desired_cursor == .None) {
-            sdl.show_cursor(i32(sdl.Bool.False));
+            sdl.ShowCursor(0);
         } else {
             chosen_cursor := state.cursor_handles[imgui.Mouse_Cursor.Arrow];
             if state.cursor_handles[desired_cursor] != nil {
                 chosen_cursor = state.cursor_handles[desired_cursor];
             }
-            sdl.set_cursor(chosen_cursor);
-            sdl.show_cursor(i32(sdl.Bool.True));
+            sdl.SetCursor(chosen_cursor);
+            sdl.ShowCursor(1);
         }
     }
 }
 
 update_display_size :: proc(window: ^sdl.Window) {
     w, h, display_h, display_w: i32;
-    sdl.get_window_size(window, &w, &h);
-    if sdl.get_window_flags(window) & u32(sdl.Window_Flags.Minimized) != 0 {
+    sdl.GetWindowSize(window, &w, &h);
+    if sdl.GetWindowFlags(window) & u32(sdl.WindowFlags.MINIMIZED) == 0 {
```

Comment:
Now I suck at bit operations but changing the condition from `!=` to `==` should be wrong?

Reply:
This "appears to work" but I think it's actually because WindowFlags.MINIMIZED == 6 when the mask should be 1 << 6.